Sofa/Component/Constraint/Lagrangian/Correction/src/sofa/component/constraint/lagrangian/correction/GenericConstraintCorrection.h

```diff
@@ -72,8 +93,8 @@ class SOFA_COMPONENT_CONSTRAINT_LAGRANGIAN_CORRECTION_API GenericConstraintCorre
     void applyMotionCorrection(const core::ConstraintParams* cparams, core::MultiVecCoordId xId, core::MultiVecDerivId vId, core::MultiVecDerivId dxId,
                                core::ConstMultiVecDerivId correction, double positionFactor, double velocityFactor);
 
-    core::behavior::OdeSolver* m_ODESolver;
-    std::vector< core::behavior::LinearSolver* > m_linearSolvers;
+    sofa::core::behavior::OdeSolver* m_ODESolver = nullptr;
+    sofa::core::behavior::LinearSolver* m_linearSolver = nullptr;
```

Comment:
You don't need those two variables since you have the links

Reply:
The check is still done in the init function. The behavior would equivalent. The link is valid or not, like the pointer is null or not.